src/component-queries/convert-data.ts

```diff
@@ -105,6 +105,7 @@ export const convertFrontmatterToNormalCellLines = ({
         healthCertificate: "",
         orderLink: cellLineNode.frontmatter.order_link,
         orderPlasmid: cellLineNode.frontmatter.donor_plasmid,
+        thumbnailImage: cellLineNode.frontmatter.thumbnail_image,
```

Comment:
I would unpack this all the way, so thumbnailImage type is `gatsbyImageData | null`